src/format.rs

```diff
@@ -281,6 +281,11 @@ impl FormatBuilder {
         FormatBuilder { format: Box::new(format) }
     }
 
+    /// Consumes the builder and returns the generated `TableFormat`
+    pub fn into_table_format(self) -> TableFormat {
+        *self.format
+    }
```

Comment:
It has sense to keep dedicated methods if they return something else, say, `Result<TableFormat>` because conversion can have errors, but this is not the case. Otherwise they are longer, have the same meaning as the standard ways to do conversion and bloat the API.

The only advantage I see in these methods is that they are easier to find in docs (for now, don't know if `rustdoc` will make Table of Contents or something else to ease the navigation between own and trait methods).